Vc/common/subscript.h

```diff
@@ -233,7 +233,7 @@ struct IndexVectorSizeMatches<MinSize,
 
 template <std::size_t MinSize, typename T, std::ptrdiff_t N>
 struct IndexVectorSizeMatches<MinSize, Vc::Common::span<T, N>, false>
-    : public std::integral_constant<bool, (N == -1 || MinSize <= N)> {
+    : public std::integral_constant<bool, (N == -1 || MinSize <= static_cast<std::ptrdiff_t>(N))> {
```

Comment:
Isn't `N` already of type `std::ptrdiff_t`? Why is the cast needed?

Reply:
I made a mistake here, the purpose of this is to resolve the C4018 on the comparison above.